frontend/controller/actions/group.js

```diff
@@ -316,7 +316,9 @@ export default (sbp('sbp/selectors/register', {
   // action if we haven't done so yet (because we were previously waiting for
   // the keys), or (c) already a member and ready to interact with the group.
   'gi.actions/group/join': async function (params: $Exact<ChelKeyRequestParams> & { options?: { skipUsableKeysCheck?: boolean; skipInviteAccept?: boolean } }) {
-    sbp('okTurtles.data/set', 'JOINING_GROUP-' + params.contractID, true)
+    if (!params.options?.skipInviteAccept) {
```

Comment:
Why was this change added?

And I feel like this `skipInviteAccept` must have been removed for some reason, maybe @corrideat can also comment on that?

Reply:
Actually, `JOINING_GROUP-params.contractID` was designed to be used in `sideEffect` functions only when joining group, not when logging in. It provide us the way to ignore removing group contract if we meet `removeOurselves` while joining group. `skipInviteAccept` is true only when login.

Having investigated the `gi.actions/group/join` function and login process, I have noticed that we sync the group contracts many times while logging in. And it's not easy to decide when should set this `JOINING_GROUP-params.contractID` true. If we remove the `if` conditional statement, there won't be any errors, but the contract would be removed and synced which is redundant activity.
At the moment, I am not sure when should I set, which conditional check would be correct, so I will remove this `if` conditional statement.

However, this `JOINING_GROUP-params.contractID` should be set `true` only when we sync the group contract to join.